game/lib/challenges/challenges_page.dart

```diff
@@ -136,18 +106,96 @@ class _ChallengesPageState extends State<ChallengesPage> {
                     ],
                   ),
                 ),
-                Expanded(
-                  child: ListView.separated(
-                    padding: const EdgeInsets.all(0),
-                    itemCount: cells.length,
+                Expanded(child: Consumer4<EventModel, GroupModel, TrackerModel,
```

Comment:
It looks like this code is a repeat of what the master branch has. How is this different than the code we have currently?

Reply:
It isn't. I just forgot to add documentation and I didn't create a new branch.... oops